src/goPath.ts

```diff
@@ -11,16 +11,21 @@
 import fs = require('fs');
 import path = require('path');
 import os = require('os');
+import vscode = require('vscode');
 
 let binPathCache: { [bin: string]: string; } = {};
 let runtimePathCache: string = '';
 
 export function getBinPathFromEnvVar(toolName: string, envVarValue: string, appendBinToPath: boolean): string {
-	toolName = correctBinname(toolName);
+	let binname = correctBinname(toolName);
```

Comment:
`correctBinName` gets run twice resulting in strings like `golint.exe.exe` in Windows